cmd/erasure-object.go

```diff
@@ -703,10 +707,9 @@ func (er erasureObjects) putObject(ctx context.Context, bucket string, object st
 }
 
 func (er erasureObjects) deleteObjectVersion(ctx context.Context, bucket, object string, writeQuorum int, fi FileInfo) error {
+	defer ObjectPathUpdated(path.Join(bucket, object))
```

Comment:
```suggestion
	defer ObjectPathUpdated(pathJoin(bucket, object))
```

To preserve the object `/` at at the end

Reply:
It doesn't really matter. It is always stripped anyway. Bloom filter doesn't distinguish.